### PR TITLE
db/dashboard-component

### PR DESCRIPTION
**Pull request by karolkolodziej**
Fixed proj4js warning.

### CODE REVIEW

**Comment by bre1470** [comment]
I do not see any change, except code style. What was the warning about?

**Comment by karolkolodziej** [comment]
@bre1470 The warning was that the `proj4js` is missing.
This is the most relevant change:
https://github.com/highcharts/highcharts/pull/18164/files#diff-7286a1061867ff6e38ddb7a8546872c01ca1d2e2dac6127a74980ba03347e529R106

**Review by pawellysy — APPROVED** [review]
LGTM

File: samples/dashboard/demos/dashboard-component/demo.mjs [hunk 1]
```diff
@@ -1,165 +1,167 @@
 import Highcharts from 'https://code.highcharts.com/es-modules/masters/highmaps.src.js';
-
 import HighchartsComponent from 'https://code.highcharts.com/es-modules/Extensions/DashboardPlugins/HighchartsComponent.js';
 import DOMComponent from 'https://code.highcharts.com/es-modules/Dashboard/Component/HTMLComponent.js';
 import CSVStore from 'https://code.highcharts.com/es-modules/Data/Stores/CSVStore.js';
 
 Highcharts.AST.allowedAttributes.push('rows');
 
 const style = {
```
Comment (sebastianbochan) [inline samples/dashboard/demos/dashboard-component/demo.mjs:8]: Do we need it? It's simple so we could add it in a component's config.

**Review by bre1470 — APPROVED** [review]
Looks good. 👍

File: samples/dashboard/demos/dashboard-component/demo.mjs (hunk 1 above)
Comment (karolkolodziej) [reply samples/dashboard/demos/dashboard-component/demo.mjs:8]: I added a task to asana that will be about cleaning each demo after everything is ready.

**Pull request merged**